I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 47, in script
    start music "audio/intro-story.mp3"
Exception: Sayer 'start' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 47, in script
    start music "audio/intro-story.mp3"
  File "D:\PROGRAMAS\renpy\renpy-8.0.0-sdk\renpy\ast.py", line 861, in execute
    who = eval_who(self.who, self.who_fast)
  File "D:\PROGRAMAS\renpy\renpy-8.0.0-sdk\renpy\ast.py", line 754, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'start' is not defined.

Windows-10-10.0.18362 AMD64
Ren'Py 8.0.0.22062402
VlastVN 1.0
Sun Jul  3 20:48:11 2022
